### DISCUSSION TITLE
Improve emoji designs and dimensions

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#4308] [Other Feature Feedback, Questions, & Ideas]
## The issues
Emojis are a nice thing to have and use on GitHub.
With that said do they suffer from several issues that could need some changes and improvements, which I list below.

## Unreliable designs
The Emojis seem to be just the bare-bone unicode emojis which come with the downside, that their look depends on the OS you are using.

Here is a small comparison of viewing #4085 with Edge on Windows 10 and through the official GitHub Android app.

- **Windows**
  ![windows_example](https://user-images.githubusercontent.com/11576465/123328747-40644800-d53c-11eb-849d-f3e56291d678.png)
- **Android**
  ![android_example](https://cdn.discordapp.com/attachments/412716812761825290/857720481426440222/SmartSelect_20210624-223432_GitHub.jpg)

As you can see are the emojis displaying in different styles, which are both based on the OS you're currently using.
While for some this might not be that big of a deal is it for me quite frustrating, since I spend most time on GitHub through the browser on my Windows PC and some Emojis just don't have a proper, good looking render for it.
Just look at the country flags to see what I mean:

![image](https://user-images.githubusercontent.com/11576465/123329176-cf716000-d53c-11eb-8577-df7fdf2c724c.png)

I would now suggest to use a unified Emoji set such as Twemoji which would help in making sure that all emojis look the same across all platforms without suffering from quality loss.

## Inconsistent widths
The emojis have quite inconsistent widths which is another issue you can encounter quite often. A prime example for this is the categories of discussion which all require an emoji to be displayed.

Let's take the categories of this repository as an example:
![image](https://user-images.githubusercontent.com/11576465/123329507-2f680680-d53d-11eb-9241-083b4fdce376.png)

As you can see is the "Mobile Feedback" text much further to the left than all the other category names. In addition does the "General Feedback" also have a slight indeed to the right.

For someone like me who really tries to make things look nice in Readmes and documentation is this quite an eye-sore and is triggering me a lot.

The emojis here should keep a consistent width to not have those wanky alignments.

## Final words
I hope this can be looked into. I know those are rather minor issues, but they make GitHub look worse than what it really is.

**Comment by Levi-Lesches** [comment]
I don't think GitHub should be designing "canon" emojis, it's just not the right place for it to be done. I agree that different operating systems having different sets of emoji can be annoying, but that should be fixed at the OS level. Imagine if every popular app had their own set of emojis -- that'd be extremely redundant.

**Reply by Andre601** [reply]
> I agree that different operating systems having different sets of emoji can be annoying, but that should be fixed at the OS level

While I can agree that emojis should be fixed in the OS level, is this also not that easy. That's essentially like saying "use a bucket of water to extinguish a forest fire".
Microsoft is large and my feedback would have little to no exposure to make them notice it at all. And not to mention that it isn't even clear *where* you would share such a feedback to begin with.

It's Imo much easier to instead ask the site/app in question to just use a specific Emoji Set.

> Imagine if every popular app had their own set of emojis -- that'd be extremely redundant.

I never said anywhere that GitHub would need to make their own Emoji Set. I already suggested the usage of [Twemoji](https://twemoji.twitter.com) an Open Source (meaning they could alter emojis or even add their own like :octocat:) and comparatively neutral Emoji Set.
When you look at specific Emojis like [Mobile Phone](https://emojipedia.org/mobile-phone/) will you notice, that they are very brand-focused. Windows show a Lumia, Apple shows an iPhone.
Twemoji would be a fairly neutral option to choose and it's already used by larger sites and apps like [Discord](https://discord.com) where it helps to display the same Emoji across different Operating Systems.

We can't deny here that some Emojis (mainly the flag ones) look really odd on certain OS and that using a specific (neutral) Emoji Set would help in making clear what is seen.
Because while you may see 🇨🇭 as a flag (maybe) do I see it as "ch" which doesn't look appealing at all. This OS-Bound setup forces me to either not use it, because it's shit, use a (bad) alternative like a PNG or use and hate it because it (again) looks shit...

I get your points but also think they're not really justified. Just saying "Oh. This would make GitHub *reinvent* Emojis" is not really something.
I mean by the logic you give here should you complain about why GitHub is making and using their own SVG-Icons ([Octicons](https://primer.style/octicons)) when they could just use FontAwesome or any other existing Icon Set?
